activiti-steps/activiti-steps-assertions-api/src/main/java/org/activiti/steps/matchers/ProcessInstanceMatchers.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.steps.matchers;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.activiti.api.process.model.ProcessInstance;
+import org.activiti.api.process.model.events.ProcessRuntimeEvent;
+import org.activiti.api.process.runtime.events.ProcessCompletedEvent;
+import org.activiti.api.process.runtime.events.ProcessCreatedEvent;
+import org.activiti.api.process.runtime.events.ProcessStartedEvent;
+import org.activiti.api.task.model.Task;
+import org.activiti.steps.TaskProvider;
+
+import static org.activiti.api.process.model.events.ProcessRuntimeEvent.ProcessEvents.PROCESS_STARTED;
+import static org.assertj.core.api.Assertions.*;
+
+public class ProcessInstanceMatchers {
+
+    private ProcessInstanceMatchers() {
+    }
+
+    public static ProcessInstanceMatchers processInstance() {
+        return new ProcessInstanceMatchers();
+    }
+
+    public OperationScopeMatcher hasBeenStarted() {
```

Comment:
@erdemedeiros so here suspended and running are missing right?

Reply:
yes, suspended and running are not covered yet.